pkg/kops/kops.go

```diff
@@ -41,33 +43,50 @@ func (k *Wrapper) topologyOf(s *ec2.Subnet) api.SubnetTopology {
 }
 
 // UseVPC finds VPC and subnets that give kops cluster uses and add those to EKS cluster config
-func (k *Wrapper) UseVPC(ec2API ec2iface.EC2API, spec *api.ClusterConfig) error {
+func (k *Wrapper) UseVPC(ctx context.Context, ec2API awsapi.EC2, spec *api.ClusterConfig) error {
 	spec.VPC.CIDR = nil // ensure to reset the CIDR
 
-	allSubnets, err := aws.ListSubnets(k.cloud, k.clusterName)
+	clusterTag := fmt.Sprintf("kubernetes.io/cluster/%s", k.clusterName)
+	output, err := ec2API.DescribeSubnets(ctx, &ec2.DescribeSubnetsInput{
+		Filters: []ec2types.Filter{
+			{
+				Name:   aws.String(fmt.Sprintf("tag:%s", clusterTag)),
+				Values: []string{k.clusterName},
```

Comment:
Is this correct? Previously it was checking if the value is "owned". Now it's checking if the value is the clusterName. Did that change?

Reply:
Good catch. The value should be `owned` here.

Reply:
🎉